x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/CertificateToolTests.java

```diff
@@ -419,6 +418,9 @@ public void testGeneratingSignedPemCertificates() throws Exception {
     }
 
     public void testHandleLongPasswords() throws Exception {
+        assumeFalse("JDK bug JDK-8266279, https://github.com/elastic/elasticsearch/issues/75571",
```

Comment:
Should the linked issue be #75952? Or are you using a single issue to track them all? Might be worthwhile to close all others if this is the case. But since the mute is specific to a particular version, I think we can just close the issues.

Reply:
I think it's better to link them all to a single issue. Perhaps we should go back and change the description of the issue.

Tracking them in multiple issues makes it harder to handle once the next JDK build comes out and they all start failing (because the bug wasn't fixed) or we can remove these checks (if it is fixed).